backend/pkg/dbstore/report_group.go

```diff
@@ -13,79 +13,134 @@ type ReportGroup struct {
 	Description string `json:"description"`
 }
 
+func ReportGroupFields() string {
+	return "\n{\n\tid string" +
+		"\n\tname string" +
+		"\n\tdescription string\n}"
+}
```

Comment:
Same as above, although now now sure if I prefer the string literal... 🤷‍♂️ .

Reply:
I dunno - it was just the easiest way for me to quickly get something done.. 🤷